pages/09.Companies/docs.en.md

```diff
@@ -33,6 +33,18 @@ Above the informations of the company (name, address, and all you custom company
 
 Note also that since 3.1 it is possible to add the column 'date added to company' to reports.
 
+### Company duplicates
+
+Company name field is mark as unique identifier by default. But you can choose any other company fields as unique identifier in [custom fields][custom fields]. 
+
+In Configurations > Company Settings you can choose operator for the merge companies by the unique identifiers.
+
+![Setup operator for find duplications algorithm](company-duplicates-configuration-operator.png "Setup operator for find duplications algorithm")
+
+These settings use algorithm to find and merge duplicates companies during the import, [integrations framework][integration-framework] and other parts of Mautic
```

Comment:
```suggestion
These settings allow Mautic to find and merge duplicate companies during the import, using the [integrations framework][integration-framework] and in other parts of Mautic.
```

Reply:
I wonder if it would be helpful to be explicit about where this is being invoked?